recipes/kcp/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class KcpConan(ConanFile):
+    name = "kcp"
+    description = "A Fast and Reliable ARQ Protocol"
+    topic = ("conan", "kcp", "ARQ", "TCP", "protocol")
+    homepage = "https://github.com/skywind3000/kcp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("kcp does not support shared builds on Windows")
```

Comment:
Any reason to not enable WINDOWS_EXPORT_ALL_SYMBOLS in cmake wrapper?

Reply:
That should work fine. 
I opened https://github.com/skywind3000/kcp/pull/254 but it does not like they review and/or accept pr's in a timely manner.